Shoko.Server/Settings/ServerSettings.cs

```diff
@@ -167,13 +158,16 @@ public static void LoadSettings()
         private static ServerSettings LoadLegacySettings()
         {
             Legacy.LoadSettings();
+            NumberFormatInfo dotDecSym = new NumberFormatInfo() {
+                NumberDecimalSeparator = "." // ignore locale decimal symbol and always use dot
+            };
             var settings = new ServerSettings()
             {
                 ImagesPath = Legacy.ImagesPath,
                 AnimeXmlDirectory = Legacy.AnimeXmlDirectory,
                 MyListDirectory = Legacy.MyListDirectory,
                 ServerPort = (ushort)Legacy.JMMServerPort,
-                PluginAutoWatchThreshold = double.Parse(Legacy.PluginAutoWatchThreshold),
+                PluginAutoWatchThreshold = double.Parse(Legacy.PluginAutoWatchThreshold, dotDecSym),
```

Comment:
You should just be using https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.invariantculture?view=netframework-4.8